### DISCUSSION TITLE
[Feature] Option to include custom folders for backup

### DISCUSSION THREAD

**Discussion by kazink** [immich-app/immich#10862] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi, I would like to request a feature to be able to define custom folders for backup in addition to the albums.

I am using albums already, but not all folders I would like to back up are recognised by Immich as albums. And if something hasn't been recognised as an album, there is no option in Immich to back it up (or I couldn't find it).

The two folders I would like to back up, are folders WhatsApp uses to store received photos and videos:
/storage/emulated/0/WhatsApp/media/WhatsApp Video/Private
/storage/emulated/0/WhatsApp/media/WhatsApp Images/Private.

The hardware I am trying this on is an Android 9 phone.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1678

**Reply by kazink** [reply]
> #1678

Hi bo0tzz, I don't think the discussion you referenced is the same as mine. 

That discussion is about having Immich preserve the album information on uploaded media.

My request is about being able to select folders for backup, that are **not** currently recognised as albums by Immich, thus cannot be uploaded **at all**.

Could you please reopen it (unless it's really a duplicate of another existing issue)?

**Reply by bo0tzz** [reply]
#7075

**Reply by kazink** [reply]
This one seems to be the same. thanks. I have searched for similar issues, but didn't find this one.
